files/zh-cn/notes_03.json

```diff
@@ -0,0 +1,4 @@
+{
+  "title": "Day 3:",
+  "description": "勒索已经开始了几天了。开头的过程仿佛显得无害，因为他只是希望我滥用政府文件的访问权和招揽聪慧的人才。这个情形开始变得吓人了。我不知道怎么摆脱这个局面。我会试着找到一个解决方案的。"
```

Comment:
I think there's a mistranslation of the word "intelligence"

Reply:
I used to regard the "intelligence" as "talents" in the script. For this part, actually I have difficulty in understanding what this word refers to for this sentence, can you expand the explanation a little bit?

Reply:
Here, intelligence is used to means information, in a spying context. So instead of
> It seemed harmless at first, he only wanted me to abuse my access to the State files and gather intelligence.

I could have write

> It seemed harmless at first, he only wanted me to access government files and insider informations.

I don't know if I'm clear 😅

Reply:
Oh then I can understand what it means. 

"Insider Information" can be translated into "内部消息/内幕消息", the difference is, the latter can be more about conspiracy haha. "招揽聪慧的人才" is totally wrong and should be changed to "搜集内幕消息".

Reply:
Indeed, your last proposal looks fine !